selfdrive/navd/map_renderer.cc

```diff
@@ -160,6 +160,13 @@ void MapRenderer::update() {
   gl_functions->glFlush();
   double end_t = millis_since_boot();
 
+  while (TEST_MODE && !loaded()) {
```

Comment:
why is this only needed in test mode? seems like this is useful to have while running live

Reply:
might be best to have this running live as well, just need to make sure the loop isn't too tight